app/content/drill/[id]/attempts/[attempt].js

```diff
@@ -1,190 +1,26 @@
-            <View style={styles.chartSection}>
-              <Text style={styles.sectionTitle}>Shot Tendency</Text>
-              <View style={{ ...styles.chartContainer, width: width * 0.8 }}>
-                <ScatterChart
-                  style={styles.chart}
-                  backgroundColor="#ffffff"
-                  data={[
-                    {
-                      color: "blue",
-                      unit: "%",
-                      values: dots,
-                    },
-                  ]}
-                  horizontalLinesAt={[0]}
-                  verticalLinesAt={[0]}
-                  minY={-10}
-                  maxY={10}
-                  minX={-35}
-                  maxX={35}
-                  chartWidth={width * 0.8}
-                />
-              </View>
-            </View>
-          )}
-
-          {attempt["shots"] &&
-            attempt["shots"].map((shot) => (
-              <ShotAccordion
-                key={shot["sid"]}
-                shot={shot}
-                drillInfo={drillInfo}
-                total={numTrunc(attempt["shots"].length)}
-              />
-            ))}
-        </ScrollView>
-      </SafeAreaView>
-    </>
+    <SafeAreaView style={{ flex: 1 }} edges={["right", "top", "left"]}>
+      <Appbar.Header style={{ backgroundColor: "FFF" }} statusBarHeight={0}>
+        <Appbar.BackAction onPress={navigation.goBack} color={"#F24E1E"} />
+        <Appbar.Content title="Drill Results" titleStyle={styles.title} />
+      </Appbar.Header>
+      <ResultScreen drillId={drillId} attemptId={attemptId} />
+    </SafeAreaView>
   );
 }
 
-export default Result;
-
 const styles = StyleSheet.create({
-  container: {
-    padding: 20,
-    paddingBottom: 0,
-  },
-  header: {
-    flexDirection: "row",
-    justifyContent: "space-between",
-    marginBottom: 20,
-  },
-  headerText: {
-    fontSize: 18,
-    fontWeight: "bold",
-  },
-  sectionTitle: {
-    fontSize: 26,
-    fontWeight: "bold",
-    marginBottom: 10,
-    alignSelf: "center",
-  },
-  dataSection: {
-    alignItems: "center",
-    justifyContent: "center",
-    marginBottom: 20,
-    padding: 15,
-    backgroundColor: "#f5f5f5",
-    borderRadius: 10,
-    width: "60%",
-    alignSelf: "center",
-  },
-  dataTitle: {
-    fontSize: 16,
-    fontWeight: "bold",
-    marginBottom: 10,
-  },
-
-  dataRow: {
-    flexDirection: "row",
-    justifyContent: "space-between",
-    alignItems: "center",
-    marginBottom: 5,
-  },
-
-  dataLabel: {
-    fontSize: 14,
-  },
-
-  dataValue: {
-    fontSize: 14,
-    fontWeight: "bold",
-  },
-
-  chartSection: {
-    marginBottom: 30,
-    alignItems: "center",
-  },
-  chartContainer: {},
-  chart: {
-    backgroundColor: "#ffffff",
-    position: "absolute",
-    left: 0,
-    right: 0,
-    top: 0,
-    bottom: 0,
-    alignSelf: "center",
-    height: 350,
-  },
-
-  restartButton: {
-    margin: 10,
-    alignItems: "center",
-  },
+  fontSize: 18,
+  fontWeight: "bold",
```

Comment:
Are these styles still being used? it looks like the Appbar header is using them but there is no "title" style defined